source/common/http/http3/conn_pool.h

```diff
@@ -97,6 +98,15 @@ class ActiveClient : public MultiplexedActiveClientBase {
   uint64_t quiche_capacity_ = 100;
 };
 
+// An interface to propagate H3 handshake result.
+class PoolConnectResultCallback {
+public:
+  virtual ~PoolConnectResultCallback() = default;
+
+  // Called when the handshake completes.
```

Comment:
is complete
Also think it's worth spelling out a bit more for folks who are less familiar with HTTP/3?